### PR TITLE
Multiple value support

### PR DESCRIPTION
**Pull request by AnikaAgiwal2711**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 80% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: powerflex/helper/sdc_helper.go [hunk 1]
```diff
@@ -475,37 +517,114 @@ func CheckForSDCName(system *goscaleio.System, sdcDetail models.SDCDetailDataMod
...
-				break
-			} else if obj2.SDCID.ValueString() != "" && checkID[obj2.SDCID.ValueString()] {
-				found = true
-				delete(checkIP, obj2.IP.ValueString())
-				delete(checkID, obj2.SDCID.ValueString())
-				break
+		for _, obj1 := range state {
+			found := false
+			for _, obj2 := range plan {
+				if (obj2.IP.ValueString() != "" || obj2.DataNetworkIP.ValueString() != "") && (checkIP[obj2.IP.ValueString()] || checkIP[obj2.DataNetworkIP.ValueString()]) {
```
Comment (shenda1) [inline powerflex/helper/sdc_helper.go:534]: Can we add some comments here?

File: powerflex/provider/clutser_resource.go [hunk 2]
```diff
@@ -231,6 +242,20 @@ func (r *clusterResource) ValidateConfig(ctx context.Context, req resource.Valid
 			}
 		}
 
+		if presentInPrimary == true && presentInSecondary == false {
+			resp.Diagnostics.AddError(
```
Comment (shenda1) [inline powerflex/provider/clutser_resource.go:246]: Did we add this check for the meanignful error message? I believe CSV parse API is taking care of this scenario.

Comment (AnikaAgiwal2711) [reply powerflex/provider/clutser_resource.go:246]: As krunal said, it's not taking care for this scenario.

File: powerflex/helper/sdc_helper.go (hunk 1 above)
Comment (AnikaAgiwal2711) [reply powerflex/helper/sdc_helper.go:535]: Added

**Comment by AnikaAgiwal2711** [comment]
run e2e test

**Review by shenda1 — APPROVED** [review]

**Review by Krunal-Thakkar — APPROVED** [review]

**Pull request merged**